examples/overrides/d1-tag-next/package.json

```diff
@@ -7,8 +7,8 @@
 		"build": "next build",
 		"start": "next start",
 		"lint": "next lint",
-		"build:worker": "pnpm opennextjs-cloudflare build",
-		"preview:worker": "pnpm opennextjs-cloudflare preview",
+		"build:worker": "pnpm opennextjs-cloudflare build --skipWranglerConfigCheck",
```

Comment:
Why do we `--skipWranglerConfigCheck` here?
Shouldn't we `--config wrangler.e2e.jsonc` instead ?

Reply:
The build doesn't need to know about the config, it just has a check at the end of running to see if one exists and prompts to let you create one if not